fs/xfstests.py

```diff
@@ -208,7 +209,6 @@ def setUp(self):
                 packages = list(set(packages)-set(packages_remove))
             elif self.detected_distro.name == 'rhel' and\
                     self.detected_distro.version.startswith('9'):
-                packages_remove.extend(['dump'])
```

Comment:
If .. else block is no longer needed and can be collapsed into a single if stmt.

Reply:
I have kept the rhel versioning as is.